Iceberg-Plugin-GitHub/IceGitHubPullRequestDetailModel.class.st

```diff
@@ -172,7 +172,7 @@ IceGitHubPullRequestDetailModel >> newChecksTable [
 			evaluated: [ :aStatus | aStatus at: 'description' ];
 			yourself);
 		beResizable;
-		contextMenu: [ (self rootCommandsGroup / 'checks commands') beRoot asMenuPresenter ]
+		actions: (self rootCommandsGroup / 'checks commands')
```

Comment:
Looks much better! 
I wonder if this affects Spec backends or they don't have impact to support this new API.
Have to check